ios/FluentUI/Controls/Button.swift

```diff
@@ -187,7 +195,9 @@ open class Button: UIButton {
 
         layer.borderWidth = style.hasBorders ? Constants.borderWidth : 0
 
-        contentEdgeInsets = style.contentEdgeInsets
+		if !isUsingCustomContentEdgeInsets {
```

Comment:
wouldn't this be always false because it is initialization?

Reply:
It will unless the calling code sets the contentEdgeInsets property:

    open override var contentEdgeInsets: UIEdgeInsets {
        didSet {
            isUsingCustomContentEdgeInsets = true
        }
    }

Reply:
but you need to initialize the button first before you can set the contentedgeinset of that instance?